### ISSUE TITLE
Remove all prints that are outside of RoboBuggyLogic

### ISSUE THREAD

**Issue by TrevorDecker** [CMU-Robotics-Club/RoboBuggy#355]
Ideally this will be implemented by a style check which will not pass any code that has a print outside of buggyLogic.  

This is low priority but will help with debugging

**Comment by TrevorDecker** [comment]
this is addressed by #374, currently waiting on review to be pulled into master

**Comment by VasuAgrawal** [comment]
there's currently a few other prints that are happening due to the various libraries we are using. Those should be caught and eliminated. Bing maps is a repeat offender.

**Comment by TrevorDecker** [comment]
This has been merged into master 

The core functionality has been implemented I am going to open a new issue for bing maps. 

Really the way I implemented this is a little hacky but it does work.  The intent of the issue was to stop debug statements from being added to master
